test/config/webConfig.test.js

```diff
@@ -0,0 +1,47 @@
+const chai = require('chai')
+const proxyquire = require('proxyquire')
+
+chai.use(require('sinon-chai'))
+chai.should()
+
+describe('#webConfig', () => {
+  const webConfig = proxyquire('../../config/webConfig', {
+    './app': {}
+  })
+
+  describe('Uses defaults if nothing specified', () => {
+    it('uses "/" as the default base', () => {
```

Comment:
don't need curly braces for all your assertions, removing them and implicitly returning booleans is safer for catching false positives failures

Reply:
TIL! You will have to talk to me more about this, I have never returned anything from my UT -- and now you made me think about it, and I need to see what I have been missing!